src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -148,11 +150,11 @@ static void setEntities(ngsiv2::Registration* regP, const mongo::BSONObj& cr0)
 
 /* ****************************************************************************
 *
-* setAttributes - 
+* setAttributes -
 */
-static void setAttributes(ngsiv2::Registration* regP, const mongo::BSONObj& cr0)
+static void setAttributes(ngsiv2::Registration* regP, const mongo::BSONObj& cr0, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for attributes is not a parameter but fixed by the database model.